src/lib/operators/index_scan.hpp

```diff
@@ -40,18 +39,19 @@ class IndexScan : public AbstractReadOnlyOperator {
   void _on_set_parameters(const std::unordered_map<ParameterID, AllTypeVariant>& parameters) override;
 
   void _validate_input();
-  std::shared_ptr<AbstractTask> _create_job(const ChunkID chunk_id, std::mutex& output_mutex);
-  RowIDPosList _scan_chunk(const ChunkID chunk_id);
 
  private:
-  const ChunkIndexType _index_type;
-  const std::vector<ColumnID> _left_column_ids;
+  friend class LQPTranslatorTest_PredicateNodePrunedIndexScan_Test;
```

Comment:
We usually just add the test class (not the fixture class) as friend and add a method to it that accesses the members. Sth like
```c++
friend class LQPTranslatorTest;
```
here and in the test sth like this:
```c++
class LQPTranslatorTest : public BaseTest {
  ...
  ColumnID index_scan_column_id(const std::shared_ptr<const IndexScan>& index_scan) {
    return index_scan->_indexed_column_id;
  }
}
```